test/unit/org/apache/cassandra/schema/CreateTableValidationTest.java

```diff
@@ -48,4 +51,89 @@ public void testInvalidBloomFilterFPRatio() throws Throwable
         // sanity check
         createTable("CREATE TABLE %s (a int PRIMARY KEY, b int) WITH bloom_filter_fp_chance = 0.1");
     }
+
+    @Test
+    public void testCreateTableErrorOnNonClusterKey()
```

Comment:
There is plenty of repetition. We can improve readability by having a utility method and splitting the test into a few small ones; each test name will reflect exactly what we test. Tomek already has something similar in our fork. You can check it and get inspired by it. We can mark him as then co-author on the ticket.

Reply:
+1 with @ekaterinadimitrova2